docs/ibc/Overview.md

```diff
@@ -31,28 +31,28 @@ Provider side and the Consumer side. This is done in multiple stages.
 
 Establish a channel (allow list):
 
-1. Admin deploys _Receiver_ on Consumer side.
-2. Admin deploys _External Staker_ on Provider side, registering (connection, port) from _Receiver_ contract from (1)
-3. Admin updates _Receiver_ (1) with the (connection, port) from (2)
-4. Anyone establishes a new channel between _Receiver_ and _External Staker_. They ensure the other side matches the whitelist.
-5. Neither contract accept any more channels. They only accept one channel and only what matches the allow list.
+2. Admin deploys _Converter_ and _Virtual Staking_ on Consumer side, both referencing each other.
+3. Admin deploys _External Staker_ on Provider side, registering `(connection, port)` from _Converter_ contract from (1)
+4. Admin updates _Converter_ (1) to allow the `(connection, port)` from (2)
+5. Anyone establishes a new channel between _Converter_ and _External Staker_.
+Both contracts ensure the other side matches the stored `(connection, port)` and refused the channel otherwise.
+6. Neither contract accept any more channels. They only accept one channel and only what matches their config.
 
 Now that we have a channel and know which contract is talking to whom, we need
 to authorize them:
 
 1. Admin removes their admin rights on both contracts (or passes to chain governance)
```

Comment:
Maybe worth noting that for some chains the `admin` could be chain governance?

Reply:
Not really. I mean, deploying a contract cross-chain via chain governance is a PITA.
Also, my experience doing deployments via DAO votes is this is a lot more work and harder to correct.

I propose having some setup account configure them, but then just pull in chain governance to review it and grant it access. Until it has a virtual staking allowance, there is no need for governance to be involved... it is just an arbitrary contract.

But I guess I could leave it so the "admin" could be anything you please. I was trying to show how we don't need many different voting periods to do stuff

Reply:
Also, as to deploying... we want to deploy the converter and link it to the provider chain **before** it has any capability to mint tokens. Best to involve chain governance at the end, as it only makes deploys harder to have a layer of indirection that doesn't actually grant powers.